src/Process/LayeredMaterialNodeProcessing.js

```diff
@@ -289,7 +294,7 @@ export function updateLayeredMaterialNodeElevation(context, layer, node, force)
                 terrain.texture.flipY = false;
                 terrain.texture.needsUpdate = true;
             }
-            node.setTextureElevation(terrain);
+            if (terrain.texture.image.data[0] && terrain.texture.image.data[terrain.texture.image.data.length - 1] > -99999) node.setTextureElevation(terrain);
```

Comment:
_+_ This fix goes with a rapid V2 release and can be kept in the future. 
_+_ Very few users will mix many DTM on the same region
 _-_   It will certainly need a more strong and generic approach as we mentioned above (2.1?)